Makefile

```diff
@@ -455,10 +454,6 @@ install-ci: install-test-tools install-build-tools
 test-ci: GOTEST := $(GOTEST_QUIET) -json
```

Comment:
do we still need this?

Reply:
Nope, good catch; thanks! Removed `-json` flag in: [5675e49](https://github.com/jaegertracing/jaeger/pull/5126/commits/5675e49cf86e78f9b9ce3d5b652f4fa1f4835d02)